src/UI/Dropdown.elm

```diff
@@ -1,5 +1,5 @@
 module UI.Dropdown exposing
-    ( Dropdown, BasicConfig, basic, filterable
+    ( Dropdown, BasicConfig, basic, filterable, autoCompleteHelper
```

Comment:
"autocomplete" should be a single word, also I'm not sure I get the "helper" part, if possible I'd keep it shorter